.github/workflows/rust.yml

```diff
@@ -19,26 +19,41 @@ jobs:
           default: true
           override: true
           components: clippy, rustfmt
+      - uses: Swatinem/rust-cache@v1
+        with:
+          key: ${{ runner.os }}-checkout
 
       - name: Cancel previous runs
         uses: styfle/cancel-workflow-action@0.5.0
         with:
           access_token: ${{ github.token }}
 
+      - name: Build
+        run: cargo build --all --release
+      - uses: Swatinem/rust-cache@v1
+        with:
+          key: ${{ runner.os }}-build
+
       - name: Run tests
         run: cargo test
+      - uses: Swatinem/rust-cache@v1
```

Comment:
This step will reuse cached dependencies and just recompile the application, right? Since tests aren’t compiled as a release?